images/common/install-sambacc-common.sh

```diff
@@ -1,5 +1,46 @@
 #!/bin/bash
 
+SAMBACC_DEPS_COPR="centos9@phlogistonjohn/sambacc-extras-deps"
+SAMBACC_COPR="phlogistonjohn/sambacc"
+
+ensure_copr() {
+    if ! rpm -q --whatprovides 'dnf-command(copr)' >/dev/null; then
+        dnf install -y 'dnf-command(copr)'
+    fi
+}
+
+enable_copr() {
+    ensure_copr
+
+    if [ "${CHROOT}" ]; then
```

Comment:
What is the intention behind introducing `CHROOT`?

Reply:
Yeah, to be added to the containerfile, but it avoids having to edit the script "every time"